routes/users.js

```diff
@@ -30,17 +21,23 @@ router.get("/users", async (req, res) => {
 router.get("/users/:id", async (req, res) => {
```

Comment:
Can we please change to "/users/get-user/:id"

Reply:
Discusssed in Discord channel, will keep RESTful endpoint naming